### ISSUE TITLE
[Feature] 需要高维mul/sub/div

### ISSUE THREAD

**Issue by liuwenda4** [tile-ai/tilelang-ascend#1236]
## Feature Description
A clear and concise description of the feature you'd like to request.
参考这个api介绍，需要mask逐bit模式
https://www.hiascend.com/document/detail/zh/CANNCommunityEdition/900beta2/API/ascendcopapi/atlasascendc_api_07_0037.html
mask逐bit模式
template <typename T, bool isSetMask = true>
__aicore__ inline void Mul(const LocalTensor<T>& dst, const LocalTensor<T>& src0, const LocalTensor<T>& src1, uint64_t mask[], const uint8_t repeatTime, const BinaryRepeatParams& repeatParams)
## Use Case
Describe the use case or scenario where this feature would be helpful.

## Proposed Solution
If you have a proposed solution or implementation idea, describe it here.

## Feature Type
- [ ] New operator/kernel (e.g., new GEMM variant, new attention mechanism)
- [x] New API/primitive (e.g., new T.xxx operation)
- [ ] Compiler optimization (e.g., new pass, performance improvement)
- [ ] New backend support (e.g., new hardware target)
- [ ] Developer tooling (e.g., debugging, profiling)
- [ ] Documentation improvement
- [ ] Other: [specify]

## Operator Details (if applicable)

**Operator Type:**
[e.g. GEMM, Flash Attention, Softmax, LayerNorm, Convolution, etc.]

**Input/Output Specification:**
```
Input shapes and dtypes:
- A: [M, K], float16
- B: [K, N], float16

Output shapes and dtypes:
- C: [M, N], float16
```
下面是一个测试用例，可以参考这个测试用例
def mul_mask_kernel(M, N, dtype="float16"):
    block_M = 2
    VEC_NUM = 2

    @T.prim_func
    def main(
        A: T.Tensor((M, N), dtype),  # type: ignore
        B: T.Tensor((M, N), dtype),  # type: ignore
        C: T.Tensor((M, N), dtype),  # type: ignore
    ):
        with T.Kernel(M // block_M, is_npu=True) as (cid, vid):
            a_ub = T.alloc_ub((block_M // VEC_NUM, N), dtype)
            b_ub = T.alloc_ub((block_M // VEC_NUM, N), dtype)
            c_ub = T.alloc_ub((block_M // VEC_NUM, N), dtype)

            T.copy(A[cid * block_M + vid * block_M // VEC_NUM, 0], a_ub)
            T.copy(B[cid * block_M + vid * block_M // VEC_NUM, 0], b_ub)
            T.tile.mul_mask(
                c_ub, a_ub, b_ub,
                [0xFFFFFFFFFFFFFFFF, 0xFFFFFFFFFFFFFFFF],
                N // 128,  # repeat_time: 128 elements per repeat for 16-bit
                1, 1, 1,   # blk strides
                8, 8, 8,   # rep strides
            )
            T.copy(c_ub, C[cid * block_M + vid * block_M // VEC_NUM, 0])

    return main

@pytest.mark.parametrize("dtype", ["float16"])
@pytest.mark.parametrize("target", ["ascendc"])
@pytest.mark.parametrize("shape", [(2, 512)])
def test_mul_mask(dtype, target, shape):
    M, N = shape
    func = mul_mask_kernel(M, N, dtype)
    func = tilelang.compile(func, out_idx=[-1], pass_configs=pass_configs, target=target)

    torch_dtype = torch.float16 if dtype == "float16" else torch.float32
    a = torch.randn(M, N, dtype=torch_dtype).npu()
    b = torch.randn(M, N, dtype=torch_dtype).npu()
    torch.npu.synchronize()

    c = func(a, b)
    torch.npu.synchronize()
    ref_c = a * b

    torch.testing.assert_close(c.cpu(), ref_c.cpu(), rtol=1e-2, atol=1e-2)
**Performance Requirements:**
[e.g. target throughput, latency, memory bandwidth]

**Reference Implementation (if any):**
[e.g. PyTorch implementation, CUDA kernel, paper reference]

## Alternative Solutions
Describe any alternative solutions or features you've considered.

## Implementation Constraints (if known)

**Programming Mode Preference:**
- [ ] Developer mode (automatic)
- [ ] Expert mode (manual control)
- [x] Both modes supported

**Memory Constraints:**
[e.g. L1 buffer size, UB buffer size constraints]

**Target Hardware:**
- [x] Ascend A2
- [x] Ascend A3
- [ ] Other Ascend variants

## Additional Context
Add any other context, references, or screenshots about the feature request here.

## Willingness to Contribute
- [x] I'm willing to submit a PR for this feature
- [ ] I can help with testing
- [ ] I can provide reference implementations
- [ ] I need guidance to get started

**Comment by ChaoyangJi** [comment]
Will be planned next month.

**Comment by liggest** [comment]
#1255 内提交的 `row_expand_mul/sub/div_experiment` 接口，如果传参形式可以和 `mul/sub/div` 统一，应该可以根据入参 Buffer 的形状做逻辑分支，合并入 `mul/sub/div` 接口中吧
